getting-started/windows-development-environment-trellis.md

```diff
@@ -12,8 +12,8 @@ post_date: 2019-02-21 16:05:55
 Trellis relies on a few other software tools. Install these tools:
 
 - VirtualBox >= 4.3.10 (install on Windows)
-- Vagrant >= 1.8.5 (install on Windows)
-- Ansible >= 2.4 (install in WSL)
+- Vagrant >= 1.8.5 (install on Windows and WSL)
+- Ansible >= 2.7, < 2.8 (install in WSL)
```

Comment:
We already have to duplicate these in a few places... we could just link to https://github.com/roots/trellis/blob/master/requirements.txt maybe? (even though it's not as readable)

Reply:
It would be nice if we could have a shortcode or tag that would just insert requirements pulled from a central location on view/build, but I'm not sure how the system we use to sync this stuff works/if it supports something like that.